src/components/Header.js

```diff
@@ -55,8 +79,9 @@ export default class Header extends Component {
         </button>
         <div className='collapse navbar-collapse' id='navbarSupportedContent'>
           <ul className='navbar-nav mr-auto'>
-            <li className='nav-item active'><Link className='nav-link' to='/pets'>Pets</Link></li>
-            <li className='nav-item active'><Link className='nav-link' to='/favorites'>Favorites</Link></li>
+            {this.context.isShelterAdmin
```

Comment:
The way this currently works, non-logged in users still see "My Profile" and "Favorites" page. Do we want those hidden when the user isn't logged in?

Reply:
I think we can still have them listed, the non-logged users will be redirected to the login page anyway